### DISCUSSION TITLE
Niimbot Label Printer Integration

### DISCUSSION THREAD

**Discussion by piramja** [inventree/InvenTree#8275] [Show and tell]
I added support for printing labels with Niimbot label printers: https://github.com/piramja/inventree-niimbot-plugin

The plugin is based on the code from [labbots/NiimPrintX](https://github.com/labbots/NiimPrintX/tree/main) and [LorisPolenz/NiimPrintX](https://github.com/LorisPolenz/NiimPrintX/tree/main).

You can configure printer model, density, rotation, scaling and offset in the plugin settings web interface.

Since i only have the B1 model, it needs some testing with other printer models. Feel free to report any issues or add bugfixes.

**Comment by matmair** [comment]
@piramja do you want to list this on the plugin repo?

**Reply by piramja** [reply]
Sure! How can i do this?

**Reply by matmair** [reply]
via PR to the website - a good sample is https://github.com/inventree/inventree-website/pull/195

**Reply by piramja** [reply]
Alright will do! Thanks for the heads up!
